tests/features/csvwcommandline.feature

```diff
@@ -72,3 +72,64 @@ Feature: Testing the csvw command group in the CLI
     When the pmdutils command CLI is run with "csvw find-where --negate 'ASK WHERE { ?s a <http://www.w3.org/2004/02/skos/core#ConceptScheme>. }'"
     Then the CLI should succeed
     And the CLI should print "single-measure-bulletin.csv-metadata.json"
+
+  Scenario: The pull command should automatically copy over csv-metadata.json files to ouput_dir.
```

Comment:
Perhaps we could give this a better name to make it clear thart we're testing the functionality to pull CSV-Ws from a file path? `The pull command should support pulling CSV-Ws from filesystem paths`?